src/main/java/com/example/analytics/controller/coinMarketCap/CryptoPriceController.java

```diff
@@ -0,0 +1,52 @@
+package com.example.analytics.controller.coinMarketCap;
+
+import com.example.analytics.exception.coinMarketCap.ApiException;
+import com.example.analytics.service.coinMarketCap.CryptoPriceService;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+@RestController
+@RequestMapping("/crypto/price")
+public class CryptoPriceController {
+    private final CryptoPriceService cryptoPriceService;
+
+    public CryptoPriceController(CryptoPriceService cryptoPriceService) {
+        this.cryptoPriceService = cryptoPriceService;
+    }
+
+    @GetMapping("/{symbol}/{convert}")
+    public String getCryptoPrice(@PathVariable String symbol, @PathVariable String convert) {
```

Comment:
Could you use ParamVariable instead of Path?

Reply:
it's not possible, symbol and convert values should be properly mapped from the URL path